assets/script.js

```diff
@@ -45,7 +45,7 @@ const getArchiveCards = i => {
 
   document.getElementById(`archiveObject_${i}`).onload = function() {
     const archiveObject = document.getElementById(`archiveObject_${i}`)
-    const cards = archiveObject.contentDocument.querySelectorAll('.card')
```

Comment:
@dimakatsomm Could you go into a little more detail why you think this is a bug? Are you getting an error? Are the archived cards not rendering for you? 

@Syknapse Does this code need refactored? I've seen .contentDocument used for iFrames, is that how we have this project rendering?

Reply:
Hi @lax0248917 
From my knowledge contentDocument is a property of iFrame, thus it will return null for an Object? Also, I was getting an error in my console of this returning null, and the 'Contributions so far...' count not working.

Reply:
@dimakatsomm, would you mind sharing your console logs showing the error?

Reply:
@lax0248917 here is the console and the page before my fix
![image](https://user-images.githubusercontent.com/47429454/126899148-f9b61564-910a-4403-a243-ab6202891019.png)
![image](https://user-images.githubusercontent.com/47429454/126899168-6f5cf22f-4394-4e43-96c8-07b7c80c9bf6.png)

Reply:
@dimakatsomm Thank you. Also if I might ask, what browser and version are you using. We want to make sure that the project works cross platform. Appreciate you taking your time to help identify the issue. One thing I would suggest is that when you come across things that you believe to be bugs, or just issues with the project in general that you raise an "Issue" here on github. You can do so right here 
![image](https://user-images.githubusercontent.com/48189365/126909529-d0503e19-4233-49fb-bdc0-00c0b0e039a2.png)

This will help us track these problems and adds value through documentation in the future :) Thanks again.

Reply:
@lax0248917 No problem. I am using Chrome Version 92.0.4515.107 (Official Build) (64-bit). Cool, in the future I will definitely use the issue tab.

Reply:
@dimakatsomm I am using the same build of Chrome and this is my console at page load
![image](https://user-images.githubusercontent.com/48189365/127001370-f41a4823-fa91-4d4f-9855-e09534443fef.png)
Any chance you might have an extension installed that is causing an issue?